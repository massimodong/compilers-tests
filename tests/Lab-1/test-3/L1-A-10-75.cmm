int OMBs5x [ 560280141 ] %, pG4p9nHX94OQVs 

;
int ;
struct {
  float MnkvBC07ElU9ZrzbgEK = YF28ANuW || WOVti16t 
    && oHhszOQfZyKKGv_IWiTR >= PRJLP8yqRX1VcYpDfX + XFy4_85_dUIoJbxrce 
    / ! BInW ( xWPESNr8cLQz , JXaDlDh0k98xhCliIz = KGwX3uP 
    %= cNWG3o4D > - McCNQO0ea [ D9xy - - MV ( Dl7GFL44hFCwQDeJ4l 
    = qRtrFCLICN == ! ( Zm3Esc7iNvwR8EV = EfeEpZ20gIdN9Ww 
    <= - - - BGpx4pkwLiXfrxJ8Ypl ( LLESDTpWonKE_vV7HB 
    != L5L ) / ! FG26yg0Q1g4AYK2NLTU < 1213765918 ) ) 
    - 1595555404.1380408138 * - c ( ) ] . yahJDL5atlqeC 
    ) , Q3tgUq6vcJ1JT_auz ;
}heSsxD ( ) {
}|| 
int ;
struct G6McDyl9I0oqzUZXuei ;
int ;
int EzFhxIaBz5 ( int VwJkXGD , int dPwZ_0BwL0JSziCP 
  ) {
  return BLLVqVOa1CLq ;
;}
struct {
  int IyhMh_ ;
  int QsgXVThw ;
  int PwpomwTpc5dgDpb ;
  struct lJEryhM != {
  }PBUv9H ;
  int smDrf ;
  int xKba2 bMvhWCfnqjiL ;
  int XeLG4h_f0UswbPVvuO7 ;
};
int R_mZw4ffCu3jRvFQ ( ) {
  if ( ZBvd8hBR )   n_hLNdus3mAnsLjREde ;
else   if ( _w )   if ( T )   while ( iF0K )   dh76U34PCyGbOr5EV5kW 
    ;

else   {
  }, 



}
int ;
int ;
