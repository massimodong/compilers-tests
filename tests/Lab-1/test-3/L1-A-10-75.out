Error Type A at Line 1
Error Type A at Line 9
Error Type B at Line 16
Error Type B at Line 23
Error Type B at Line 28
Error Type B at Line 31
Error Type B at Line 40
